File: Sereja and Dima.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    //freopen("input.txt","r",stdin);
    int n,a,m;
    vector<int>l,s,d;
    cin >> n;
    for (int i = 0; i < n ; ++i)
    {
        cin >> a;
        l.push_back(a);
    }

    while(l.size()){
        if(l.size() == 1)
        {
            m = max(l[0],l[l.size()-1]);
            s.push_back(m);
            l.erase(find(l.begin(),l.end(),m));
            break;
        }
        m = max(l[0],l[l.size()-1]);
        s.push_back(m);
        l.erase(find(l.begin(),l.end(),m));
        m = max(l[0],l[l.size()-1]);
        d.push_back(m);
        l.erase(find(l.begin(),l.end(),m));
    }

    int sums = accumulate(s.begin(),s.end(),0);
    int sumd = accumulate(d.begin(),d.end(),0);
    cout << sums << " " << sumd << "\n";
    return 0;
}

File: Sereja and Dima.py
n = int(input())

l = [int(x) for x in input().split()]
s, d = [], []
while(len(l)):
    if len(l) == 1:
        c = max(l[0], l[-1])
        s.append(c)
        l.pop(l.index(c))
        break
    c = max(l[0], l[-1])
    s.append(c)
    l.pop(l.index(c))
    c = max(l[0], l[-1])
    d.append(c)
    l.pop(l.index(c))

print(sum(s), sum(d))
